addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2406,6 +2406,10 @@ export class OdooEditor extends EventTarget {
                 this._selectTableCells(range);
                 appliedCustomSelection = true;
             }
+        } else if (selectedCells.length === 1 && selectedCells[0] === closestElement(selection.anchorNode, 'td')) {
```

Comment:
`closestElement(selection.anchorNode, 'td')` no need to write this we are already getting this value from `startTd`

Reply:
@visp-odoo Also  shouldnt we be handling the cell selection [here](https://github.com/odoo/odoo/pull/131161/commits/dd534fb3d8a0b95b65c8e802257dadb150d96530#diff-c309d19284e2e1f38a1dba1d68e0262371686595f3668a3bc0f1ca35e3189a0bR2362) which handles scenario for atleast two cell selected, can you look into it if it can be done as well?